README.md

```diff
@@ -32,4 +32,8 @@ test`.
 
 ### Deploy to production
 
-TODO: document production build
+* Set SECRET variable in .env file
+
+* Generate private key as described above into `pkey/pkey.pem`.
```

Comment:
I find this slightly confusing. Why the development and production builds ask us to put the private keys into two different locations?

- Development: `auth-server/pkey.pem`
- Production: `pkey/pkey.pem`

This can raise questions, at least to the uninitiated, like two I need two different keys, or can I use the same key for both.

Reply:
Currently we don't have dockerhub account available, so images must be built locally in the production server. In future we shouldn't need to transfer the source code (such as auth-server) directories to the production server, so in the context of prod environment there won't be `auth-server` directory. Perhaps it would be confusing if we also had `auth-server` directory in production, with very different contents compared to local env.

> do I need two different keys, or can I use the same key for both

Using same secrets/keys for both development and production is a bad idea. In local environment the handling of secrets is not critical, as we are only using data from development sources. In production we have real data and real users, so we should treat the secrets with caution, just like passwords etc.
An example attack scenario could be something like this: attacker figures that device calibration is signed, and they need a private key for messing with the calibration themselves. They find a slack invite link from linked google docs documents, and do a quick search in the message history, and they find a discussion where private key was shared via slack message. They can then make own calls into the calibration topic and sabotage the system.